src/components/SmallCard.jsx

```diff
@@ -1,15 +1,15 @@
-export default function Smallcard({ image, name, role }) {
+import GetImage from './GetImage';
+
+export default function Smallcard({ image, name, role, fallBack }) {
```

Comment:
```suggestion
export default function Smallcard({ image, name, role, fallback }) {
```

fallback en minuscule